src/main/java/leo/server/ServerJanitor.java

```diff
@@ -34,43 +33,46 @@ public ServerJanitor(Server server) {
         runner.start();
     }
 
-
     /////////////////////////////////////////////////////////////////
     // The main loop
     /////////////////////////////////////////////////////////////////
     public void run() {
         // Declaration
-        Vector players;
+        Vector<Player> players;
         Player sentPlayer;
-        // Loop indefinately, checking idle users
+        // Loop indefinitely, checking idle users
         int cycle = 0;
         int timer;
         while (true) {
             try {
                 players = server.getPlayers();
-                for (int i = 0; i < players.size(); i++) {
-                    Player player = (Player) players.elementAt(i);
+                for (Player player : players) {
                     player.getUser().checkIdle();
```

Comment:
The `checkIdle()` in `User.java` thing is what we want to change - have a look at that, especially to extract the numbers in the loops as constants.